tests/nlu/training_data/test_training_data.py

```diff
@@ -482,35 +489,30 @@ def test_training_data_conversion(
     #     f.write(td.as_json(indent=2))
 
 
-def test_get_file_format():
-    fformat = get_file_format("data/examples/luis/demo-restaurants_v5.json")
-
-    assert fformat == "json"
-
-    fformat = get_file_format("data/examples")
-
-    assert fformat == "json"
-
-    fformat = get_file_format("examples/moodbot/data/nlu.md")
-
-    assert fformat == "md"
+@pytest.mark.parametrize(
+    "data_file,expected_format",
+    [
+        ("data/examples/luis/demo-restaurants_v5.json", JSON),
+        ("data/examples", JSON),
+        ("examples/moodbot/data/nlu.md", MARKDOWN),
+        ("data/rasa_yaml_examples", RASA_YAML),
+    ],
+)
+def test_get_supported_file_format(data_file: Text, expected_format: Text):
+    fformat = get_file_format(data_file)
+    assert fformat == expected_format
 
-    with pytest.raises(AttributeError):
-        get_file_format("path-does-not-exists")
 
+@pytest.mark.parametrize("data_file", ["path-does-not-exists", None])
+def test_get_non_existing_file_format_raises(data_file: Text):
     with pytest.raises(AttributeError):
-        get_file_format(None)
+        get_file_format(data_file)
 
 
 def test_guess_format_from_non_existing_file_path():
     assert guess_format("not existing path") == UNK
```

Comment:
this test could be added to the parametrized one: `test_get_supported_file_format()`

Reply:
not really, if the path doesn't exist, the `get_file_format()` raises.